src/pom.xml

```diff
@@ -2315,7 +2315,7 @@
               <fork>false</fork>
               <compilerArgs>
                 <arg>-XDcompilePolicy=simple</arg>
-                <arg>-Xplugin:ErrorProne -XepExcludedPaths:${project.build.directory}/generated-sources/.* ${errorProneFlags}</arg>
+                <!--<arg>-Xplugin:ErrorProne -XepExcludedPaths:${project.build.directory}/generated-sources/.* ${errorProneFlags}</arg>-->
```

Comment:
This one, and similar in L2352, will be reverted. I committed them by accident, sorry. (This is related to compilation bug found in lntellij Idea).